<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DataDlg</class>
 <widget class="QWidget" name="DataDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>810</width>
    <height>533</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>检测记录查询</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#DataDlg{
background-color: rgb(192, 192, 192);
border-width:3px;
border-style: double;
border-color:rgb(255,255,255);
}</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>15</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>检测记录列表：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>10</y>
     <width>54</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>记录详情：</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>100</y>
     <width>450</width>
     <height>401</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>检测时间</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>区域1 颜色</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>区域2 颜色</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>40</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>检测开始时间：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="start_time">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>37</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>40</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>检测时长：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="time_total">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>37</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>40</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>分钟</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>72</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>颜色1：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="check_state">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>70</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>201</width>
     <height>451</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="check_state_2">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>70</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>72</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>颜色2：</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
